e2etests/web/regular_integration_tests/lib/text_editing_main.dart

```diff
@@ -47,11 +52,29 @@ class _MyHomePageState extends State<MyHomePage> {
               key: const Key('input'),
               enabled: true,
               controller: _controller,
-              //initialValue: 'Text1',
```

Comment:
add another test case for and without initial input?

Reply:
done. thanks for the suggestion.